test/test_public_bindings.py

```diff
@@ -50,6 +50,8 @@ def test_no_new_bindings(self):
             "AnyType",
             "Argument",
             "ArgumentSpec",
+            "Await",
```

Comment:
Is this actually supposed to be exposed as `torch.Await`? I don't see any doc for it?

Reply:
1/ Note that prototype vs private are two different things. prototype/beta/stable relates to the feature. public/private relate to the API. If something is public, it has to be documented.

2/ Yes the docstring needs to be added on the function/object and then listed (in torch.rst if it is in torch.)

Reply:
@IvanKobzarev open to other ideas but my opinion would be:
* make `torch.Await` -> `torch.jit.Await`, since users who don't need jit will probably not need to use this implementation
* make `torch._C.Await` -> torch._C._Await`, since it's basically an implementation detail / not supposed to be used by users (from what I can tell?), and that way you don't need to document it (I know `torch._C.Future` doesn't have the leading underscore, but I think that's because it was registered before this test_public_bindings check was added)

Reply:
@albanD @davidberard98 
Thanks.

Switched to `_Await` everywhere. 
The only public will be `torch.AwaitType`.